packages/example-vue/src/main.ts

```diff
@@ -11,7 +11,7 @@ async function prepare() {
 
     return worker.start({}).then(() => {
       console.groupCollapsed('[MSW] Loaded with handlers 🎉');
-      worker.printHandlers();
+      worker.listHandlers();
```

Comment:
Note that this will return an array of handlers _but you still need to print it_ to observe them. Right now, it just returns the list into the void. 

Consider adding a `console.log` here.

```js
const handlers = worker.listHandlers()
handlers.forEach(handler => console.log(handler.info.header))
```

Reply:
Or mayby even the `console.info()`

Reply:
Ah, thanks for clarifying that. If I were better at reading, I might have noticed this in the migration guide 😅